lib/appsignal_phoenix/event_handler.ex

```diff
@@ -119,11 +119,20 @@ defmodule Appsignal.Phoenix.EventHandler do
   end
 
   defp set_span_data(span, %{conn: conn} = metadata) do
+    appsignal_metadata = Appsignal.Metadata.metadata(conn)
+
     span
     |> @span.set_name_if_nil(name(metadata))
     |> @span.set_sample_data_if_nil("params", Appsignal.Metadata.params(conn))
-    |> @span.set_sample_data_if_nil("environment", Appsignal.Metadata.metadata(conn))
+    |> @span.set_sample_data_if_nil("environment", appsignal_metadata)
     |> @span.set_sample_data_if_nil("session_data", Appsignal.Metadata.session(conn))
+    |> @span.set_sample_data("metadata", %{
+      "hostname" => appsignal_metadata["host"],
```

Comment:
This is set by the agent and can't be set by the integration. It's overwritten by us.
I also don't know if we want to report the client host as metadata. We don't do that anywhere else, so I suggest we remove it.

```suggestion
```